### PR TITLE
library: improve ceph_mgr_module module

### PR DESCRIPTION
**Pull request by guits**
This moves the logic from the playbook to the ceph_mgr_module.

### CODE REVIEW

File: library/ceph_mgr_module.py [hunk 1]
```diff
@@ -112,21 +180,98 @@ def main():
...
+    cmd = []
+    out = []
+    err = []
+    rc = 0
+
+    if state in ['enable', 'disable']:
+        for m in name:
+            if m not in disabled_modules and m not in always_on_modules:
+                _cmd = generate_cmd(sub_cmd=['mgr', 'module'],
+                                    args=['disable', m],
```
Comment (guits) [inline library/ceph_mgr_module.py:204]:
```suggestion
                                    args=[state, m],
```

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by asm0deuz** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

File: library/ceph_mgr_module.py [hunk 2]
```diff
@@ -79,12 +92,67 @@
...
+
+
+def get_mgr_initial_modules(module,
+                            cluster,
+                            container_image):
+    node_name = os.uname()[1]
+    run_dir = get_run_dir(module, cluster, container_image)
+    # /var/run/ceph/ceph-mon.mon0.asok
+    socket_path = f"{run_dir}/{cluster}-mon.{node_name}.asok"
+    cmd = ['ceph', '--admin-daemon', socket_path, 'config', 'get', 'mgr_initial_modules', '--format', 'json']
```
Comment (asm0deuz) [inline library/ceph_mgr_module.py:117]:
```suggestion
cmd = generate_cmd(args=['--admin-daemon', socket_path, 'config', 'get', 'mgr_initial_modules', '--format', 'json'], cluster=cluster, container_image=container_image)
```

**Comment by guits** [comment]
jenkins test centos-container-collocation

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-container-purge

**Pull request closed without merging**